src/protocols/secure_channel/CASESession.cpp

```diff
@@ -2179,4 +2208,33 @@ System::Clock::Timeout CASESession::ComputeSigma2ResponseTimeout(const ReliableM
         kExpectedHighProcessingTime;
 }
 
+bool CASESession::InvokeBackgroundWorkWatchdog()
+{
+    bool status = false;
```

Comment:
This should probably be called "wasBlocked" or "wasStuck" or something.

Reply:
renamed to `wasBlocked`